airbyte-integrations/connectors/source-redshift/acceptance-test-config.yml

```diff
@@ -4,4 +4,26 @@ connector_image: airbyte/source-redshift:dev
 tests:
```

Comment:
Please use the latest acceptance-test-config format. [Example here.](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connectors/source-zendesk-support/acceptance-test-config.yml#L1). I'd be curious to check the test output under `high` test strictness level, we might get this for free 😄